content/manual/plugins-dev.md

```diff
@@ -119,11 +119,11 @@ a query comes in that is not a subdomain (or equal to) `example.org` the next pl
 
 Now, the world isn't perfect, and there are good reasons to "fallthrough" to the next middleware,
 meaning a plugin is only responsible for a *subset* of names within the zone. The first of these
-to appear was the *reverse* plugin that synthesizes PTR and A/AAAA responses (useful with IPv6).
+to appear was the *template* plugin that synthesizes various responses.
```

Comment:
Well...perhaps pedantic but this revises history. What about:

...was the *reverse* plugin, now replaced with the *template* plugin that synthesizes various responses.

Reply:
Thanks. Fixed.